How to set up a third party contract vendor (#20)
linuxfoundation/devex-and-commsuccess: README.md

| Merge MR | TBD | The final reviewer will generally merge the MR once one or more approvals have been received. The submitter will not generally approve their own requests. However, until we refine the process it should be fine for them to do so if the reviewers' approvals have been done. |

:information_source: Note for advanced users: members of the [Developer Experience and Community Success team](https://github.com/orgs/linuxfoundation/teams/devex-and-commsuccess-team) in the `linuxfoundation` organization in GitHub can create development branches directly under the repository where the handbook lives. These will be generally the ones that will be used to contain the changes submitted in PRs. It is not the most common development approach, but it simplifies our workflow. Working on forks and topic branches is also supported.

> How to set up a third party contract vendor (agency or individual):

Reviewer: Does these need to be a header for it to go in the TOC?

Reviewer 2: Good catch, I've added a suggestion to address this at https://github.com/linuxfoundation/devex-and-commsuccess-handbook/pull/20#pullrequestreview-1058497211

> #How to set up a third party contract vendor (agency or individual):

Reviewer 2: ```suggestion
## How to set up a third party contract vendor

These steps apply to either agencies or individuals.
```
Thanks! I'm proposing some changes:

- Add space between `#` and title for proper Markdown formatting
- Make the title a Header 2 level for consistency with the rest of the document (`##`)
- Shorten the title, move the extra info below the title.

@cmassey11 if you're happy with the changes, please feel free to accept the suggestion. Thanks!

1. The vendor either: 

>     1. Creates a scope of work (SOW) outlining what they will be doing and what it will cost. This is the preferred method. 

Reviewer 2: ```suggestion
   1. Creates a scope of work (SOW) outlining what they will be doing and what it will cost. This is the preferred method. 
```
Nested lists need to be indented at the level of the first letter of the list item above.

    2. Creates a legal agreement that includes a SOW. This method requires the LF Legal team to review the contract before approving it for LF to sign. 
2. The vendor sends either their SOW or their legal agreement to the LF internal contact, usually the Program Manager. 
3. The LF internal contact goes to [staff.linuxfoundation.org](staff.linuxfoundation.org) and opens the ‘[HR Finance Ops](https://jira.linuxfoundation.org/plugins/servlet/theme/portal/21)’ portal.
4. Inside the HR Finance Ops portal, click the ‘[Contractor & Vendor Requests](https://jira.linuxfoundation.org/plugins/servlet/theme/portal/21/create/462)’ option and fill out the form. 

>     3. In order to complete the form you will need this information from the contractor:

Reviewer 2: ```suggestion
   1. In order to complete the form you will need this information from the contractor:
```
- Fix indentation to align to previous list item
- Consider using [lazy numbering](https://google.github.io/styleguide/docguide/style.html#use-lazy-numbering-for-long-lists). In summary, you can use `1.` for all items in the numbered list.

Reviewer 2: This comment applies to all list items below, which will also need to be changed.

| Merge MR | TBD | The final reviewer will generally merge the MR once one or more approvals have been received. The submitter will not generally approve their own requests. However, until we refine the process it should be fine for them to do so if the reviewers' approvals have been done. |

:information_source: Note for advanced users: members of the [Developer Experience and Community Success team](https://github.com/orgs/linuxfoundation/teams/devex-and-commsuccess-team) in the `linuxfoundation` organization in GitHub can create development branches directly under the repository where the handbook lives. These will be generally the ones that will be used to contain the changes submitted in PRs. It is not the most common development approach, but it simplifies our workflow. Working on forks and topic branches is also supported.

> 

Reviewer 3: @cmassey11 Should this addition be a seperate document instead of in the main README?

Reviewer 2: Good point. We'll have to start planning the handbook structure sooner than later. For now, in the spirit of iteration, let's land this change. After it's in the handbook, Claire can decide whether it needs to live in a separate page.

## How to set up a third party contract vendor

These steps apply to either agencies or individuals.

1. The vendor either: 
   1. Creates a scope of work (SOW) outlining what they will be doing and what it will cost. This is the preferred method. 

>     2. Creates a legal agreement that includes a SOW. This method requires the LF Legal team to review the contract before approving it for LF to sign. 

Reviewer 2: ```suggestion
   1. Creates a legal agreement that includes a SOW. This method requires the LF Legal team to review the contract before approving it for LF to sign. 
```
Fix Markdown indentation, use [lazy numbering](https://google.github.io/styleguide/docguide/style.html#use-lazy-numbering-for-long-lists)

Reviewer 2: This was resolved after addressing the merge conflict.

2. The vendor sends either their SOW or their legal agreement to the LF internal contact, usually the Program Manager. 
3. The LF internal contact goes to [staff.linuxfoundation.org](staff.linuxfoundation.org) and opens the ‘[HR Finance Ops](https://jira.linuxfoundation.org/plugins/servlet/theme/portal/21)’ portal.
4. Inside the HR Finance Ops portal, click the ‘[Contractor & Vendor Requests](https://jira.linuxfoundation.org/plugins/servlet/theme/portal/21/create/462)’ option and fill out the form. 
   1. In order to complete the form you will need this information from the contractor:

>         1. Vendor name
>         2. Name of signatory for contract
>         3. Email address of signatory for contract
>         4. Contractor’s physical mailing address
>         5. Contractor start date
>         6. Contractor duration
>         7. Contractor type: individual or agency
>         8. New contract or extension to an existing contractor agreement (requires LF Legal review)
>         9. Standard LF contract (preferred) or 
>         10. LF department or Project the contractor will be working for
>         11. Description of work to align it with the correct budget code (ex: Marketing)
>         12. Compensation type: hourly, salary, other
>         13. Compensation amount
>         14. Compensation max budget
>         15. Name of LF internal manager
>         16. Scope of work provided by the vendor (provided as an attachment)

Reviewer 2: ```suggestion
   1. Vendor name
   1. Name of signatory for contract
   1. Email address of signatory for contract
   1. Contractor’s physical mailing address
   1. Contractor start date
   1. Contractor duration
   1. Contractor type: individual or agency
   1. New contract or extension to an existing contractor agreement (requires LF Legal review)
   1. Standard LF contract (preferred) or 
   1. LF department or Project the contractor will be working for
   1. Description of work to align it with the correct budget code (ex: Marketing)
   1. Compensation type: hourly, salary, other
   1. Compensation amount
   1. Compensation max budget
   1. Name of LF internal manager
   1. Scope of work provided by the vendor (provided as an attachment)
```

Fix Markdown indentation, use [lazy numbering](https://google.github.io/styleguide/docguide/style.html#use-lazy-numbering-for-long-lists)

Reviewer 2: This was resolved after addressing the merge conflict.

5. You will receive a notification from Jira that your form has been submitted. 
6. Your request will be reviewed by the LF Finance team to ensure the department or project’s budget has enough revenue to cover the new expense.
7. Once approved by LF Finance, The LF HR Finance Ops team will create a Docusign agreement. You will receive a notification that the agreement is “out for signature” when it is sent to the first signatory. 

>     4. The Docusign agreement is first sent to the Finance lead for that department or project for them to sign. 
>     5. It is then routed to the LF internal manager (who will manage the contractor) for them to sign.
>     6. Finally it is sent to the contract vendor for them to sign.

Reviewer 2: ```suggestion
   1. The Docusign agreement is first sent to the Finance lead for that department or project for them to sign. 
   1. It is then routed to the LF internal manager (who will manage the contractor) for them to sign.
   1. Finally it is sent to the contract vendor for them to sign.
```
Fix Markdown indentation, use [lazy numbering](https://google.github.io/styleguide/docguide/style.html#use-lazy-numbering-for-long-lists)

Reviewer 2: This was resolved after addressing the merge conflict.
